assets/schema/ensemble_schema.json

```diff
@@ -532,6 +532,7 @@
         {
           "type": "object",
           "properties": {
+            "description": "It also works in arrays",
```

Comment:
so "properties" is the special keyword in JSON Schema. Any direct children inside it will be made a property, so your description will ends up be a property named "property". Probably not what you want to do.

If you want the description on the TExtInput, put it at the root before "allOf"